src/Cardano/Shell/Presets.hs

```diff
@@ -156,7 +156,7 @@ devConfiguration =
           , pcoStaticKeyDlgCertFile     = pure Nothing
           , pcoRequiresNetworkMagic     = pure RequireNetworkMagic
           , pcoDBSerializeVersion       = pure 0
-          , pcoPBftSigThd               = pure Nothing
+          , pcoPBftSigThd               = pure 0.22
```

Comment:
Since the same value occurs in two places, why not make that a top level value and then have a comment on that value explaining what it is and why it is there.

Reply:
More generally, there is a *lot* of duplication in that file. Would it not be better so defined one of them, and then define the other as a set of modifications applied to that?